test/Claims.createDispute.ts

```diff
@@ -68,7 +70,7 @@ describe("Claims", function () {
           Claim(13),
           Amount(20),
           Side(true),
-          Expiry(38, "days"), // 30 days from the 8 days above
+          moment().add(8, "days").unix() + 2592000, // 30 days from the 8 days above
```

Comment:
This started failing out of nowhere and I have no idea why. Something is off with the `moment` library. The conditions in the `Claims` contract are ok.